app/uk/gov/hmrc/trackingconsentfrontend/controllers/AuditController.scala

```diff
@@ -23,5 +23,5 @@ import scala.concurrent.Future
 
 @Singleton
 class AuditController @Inject() (mcc: MessagesControllerComponents) extends FrontendController(mcc) {
-  def audit: Action[AnyContent] = Action.async(implicit request => Future.successful(Ok))
+  def audit: Action[AnyContent] = Action.async(Future.successful(Ok))
```

Comment:
does it need to be async? 🤷 just removed the unused request because of a warning since I removed the silencer plugin because there wasn't a version of it for the new scala version